lsp/server/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "lsp-sample-server",
```

Comment:
same in this file,  polish/clean up like author, name, description, repository ..?

Reply:
Good catch.